vcd/config_test.go

```diff
@@ -74,19 +74,24 @@ type TestConfig struct {
 		UploadPieceSize int64  `json:"uploadPieceSize,omitempty"`
 		UploadProgress  bool   `json:"uploadProgress,omitempty"`
 	} `json:"media"`
+	EnvVariables map[string]string `json:"envVariables,omitempty"`
 }
 
 const (
+	// This library major version
+	currentProviderVersion = "2.0"
```

Comment:
We have a VERSION file - it should be reused here.

Reply:
Done. Changed the code to get the version from there